examples/llm_quantization_demo.py

```
@@ -0,0 +1,47 @@
+import torch
+import matplotlib.pyplot as plt
+from inwhale.llm.formats import NF4Quantizer, FP4Quantizer
+
+def demo_llm_quantization():
+    print("=== NF4 and FP4 Quantization Demo ===")
+    
+    # 1. Create normally distributed "weights"
+    torch.manual_seed(42)
+    weights = torch.randn(1000)
+    
+    print(f"\nOriginal Weights:")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)